polkadot-parachains/statemine/src/lib.rs

```diff
@@ -498,6 +500,24 @@ pub type LocalAssetTransactor = CurrencyAdapter<
 	(),
 >;
 
+pub type FungiblesTransactor = FungiblesAdapter<
+	// Use this fungibles implementation:
+	Assets,
+	// Use this currency when it is a fungible asset matching the given location or name:
+	(
+		ConvertedConcreteAssetId<AssetId, Balance, AsPrefixedGeneralIndex<Local, AssetId, JustTry>, JustTry>,
+	),
+	// Do a simple punn to convert an AccountId32 MultiLocation into a native chain account ID:
```

Comment:
A simple "punn"?

Reply:
"unwrapping" the multilocation to get at the inner accountid, I'll remove it, it's copy-pasta